pkg/storaged/stratis-details.jsx

```diff
@@ -53,6 +54,14 @@ import { std_reply, with_keydesc, with_stored_passphrase } from "./stratis-utils
 
 const _ = cockpit.gettext;
 
+export function check_stratis_warnings(client, enter_warning) {
+    for (const p in client.stratis_pools) {
+        const blockdevs = client.stratis_pool_blockdevs[p] || [];
+        if (blockdevs.some(bd => bd.NewPhysicalSize[0] && Number(bd.NewPhysicalSize[1]) > Number(bd.TotalPhysicalSize)))
+            enter_warning(p, { warning: "unused-blockdevs" });
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230707-110927-1232dadb-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)